pkg/client/client.go

```diff
@@ -505,6 +505,13 @@ func FormatStatusResponse(w io.Writer, sr *models.StatusResponse, sd StatusDetai
 		fmt.Fprintf(w, "Proxy Status:\tNo managed proxy redirect\n")
 	}
 
+	if sr.IdentityRange != nil {
+		fmt.Fprintf(w, "Identity Range:\tmin %d, max %d\n",
+			sr.IdentityRange.MinIdentity, sr.IdentityRange.MaxIdentity)
+	} else {
+		fmt.Fprintf(w, "Identity Range:\tUnknown\n")
```

Comment:
nit: I might suggest using the words "Global identity range" here. Otherwise it may confuse folks who use `cilium identity list` and find reserved identities with low numbers (0-7 or so) or CIDR identities which set one of the upper bits and end up with 16xxxxx numbered identities.

```suggestion
		fmt.Fprintf(w, "Global Identity Range:\tmin %d, max %d\n",
			sr.IdentityRange.MinIdentity, sr.IdentityRange.MaxIdentity)
	} else {
		fmt.Fprintf(w, "Global Identity Range:\tUnknown\n")
```